src/main/java/com/example/sinitto/helloCall/service/HelloCallService.java

```diff
@@ -293,78 +288,85 @@ public void acceptHelloCallBySinitto(Long memberId, Long helloCallId) {
 
         if (recentLog.isPresent() && recentLog.get().getEndDateAndTime() == null) {
             throw new TimeLogSequenceException("이미 시작된 안부전화가 있습니다. 종료를 먼저 완료해주세요.");
         }
 
-        HelloCallTimeLog helloCallTimeLog = new HelloCallTimeLog(helloCall, sinitto);
+        HelloCallTimeLog helloCallTimeLog = new HelloCallTimeLog(helloCall, member);
         helloCallTimeLog.setStartDateAndTime(LocalDateTime.now());
 
-        HelloCallTimeLog savedHelloCallTimeLog = helloCallTimeLogRepository.save(helloCallTimeLog);
-        return new HelloCallTimeResponse(savedHelloCallTimeLog.getStartDateAndTime());
+        helloCallTimeLogRepository.save(helloCallTimeLog);
     }
 
     @Transactional
-    public HelloCallTimeResponse writeHelloCallEndTimeBySinitto(Long memberId, Long helloCallId) {
+    public void writeHelloCallEndTimeBySinitto(Long memberId, Long helloCallId) {
         HelloCall helloCall = helloCallRepository.findById(helloCallId)
                 .orElseThrow(() -> new HelloCallNotFoundException("id에 해당하는 안부전화 정보를 찾을 수 없습니다."));
 
-        Sinitto sinitto = sinittoRepository.findByMemberId(memberId)
-                .orElseThrow(() -> new SinittoNotFoundException("id에 해당하는 시니또를 찾을 수 없습니다."));
+        Member member = memberRepository.findById(memberId)
+                .orElseThrow(() -> new MemberNotFoundException("id에 해당하는 멤버를 찾을 수 없습니다."));
+
+        helloCall.checkMemberIsRightSinitto(member);
 
         HelloCallTimeLog helloCallTimeLog = helloCallTimeLogRepository
-                .findTopBySinittoAndHelloCallOrderByStartDateAndTimeDesc(sinitto, helloCall)
+                .findTopByMemberAndHelloCallOrderByStartDateAndTimeDesc(member, helloCall)
                 .orElseThrow(() -> new HelloCallNotFoundException("안부전화 로그를 찾을 수 없습니다."));
 
         if (helloCallTimeLog.getEndDateAndTime() != null) {
             throw new TimeLogSequenceException("이미 종료된 안부전화입니다.");
         }
 
         helloCallTimeLog.setEndDateAndTime(LocalDateTime.now());
-
-        return new HelloCallTimeResponse(helloCallTimeLog.getEndDateAndTime());
     }
 
     @Transactional
     public void cancelHelloCallBySinitto(Long memberId, Long helloCallId) {
         HelloCall helloCall = helloCallRepository.findById(helloCallId)
                 .orElseThrow(() -> new HelloCallNotFoundException("id에 해당하는 안부전화 정보를 찾을 수 없습니다."));
 
-        if (!sinittoRepository.existsByMemberId(memberId)) {
-            throw new SinittoNotFoundException("id에 해당하는 시니또를 찾을 수 없습니다.");
-        }
+        Member member = memberRepository.findById(memberId)
+                .orElseThrow(() -> new MemberNotFoundException("id에 해당하는 멤버를 찾을 수 없습니다."));
+
+        helloCall.checkMemberIsRightSinitto(member);
 
         helloCall.changeStatusToWaiting();
-        helloCall.setSinitto(null);
+        helloCall.setMember(null);
     }
 
     @Transactional
     public void SendReportBySinitto(Long memberId, HelloCallReportRequest helloCallReportRequest) {
```

Comment:
```suggestion
    public void sendReportBySinitto(Long memberId, HelloCallReportRequest helloCallReportRequest) {
```
대문자로 되어 있는걸 발견하였습니다!